templates/sonarqube/0/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+sonarqube-data:
+    image: busybox
+    tty: true

```

Comment:
@matisku can you add `net: none` to the data containers, also there is no need for them to stay running in the background, you may want to use `start_once` label, thanks.